types/yargs/ts4.1/index.d.ts

```diff
@@ -325,7 +325,7 @@ declare namespace yargs {
          * @param args An array of the words in the command line to complete.
          * @param done The callback to be called with the resulting completions.
          */
-        getCompletion(args: ReadonlyArray<string>, done: (completions: ReadonlyArray<string>) => void): Argv<T>;
+         getCompletion(args: ReadonlyArray<string>, done: (err: Error|null, completions: ReadonlyArray<string>) => void): Argv<T>;
```

Comment:
Should also include a promise signature ([source code](https://github.com/yargs/yargs/commit/6ad2be0cb564cbcf01f5148db05e8cf999057caf#diff-452e4f3371a75b323f89da8de1a1707b1532abef67be1a02fe7e8396e0f35020R1385)):
```suggestion
        getCompletion(args: ReadonlyArray<string>, done: (err: Error|null, completions: ReadonlyArray<string>) => void): Argv<T>;
        getCompletion(args: ReadonlyArray<string>, done?: never): Promise<ReadonlyArray<string>>;
        
```
(and add tests for the above suggestion)

Reply:
Done! Thanks for catching that 🤠